pkg/model/config/config.go

```diff
@@ -104,6 +93,28 @@ func (config *Config) AddResource(gvk GVK) bool {
 	return true
 }
 
+// ValidGroup checks that the provided group can be added to the project
+func (config Config) ValidGroup(group string) bool {
```

Comment:
The nomenclature of methods that return bool should be isX/hasX. So, IMO it should be. isGroupValid().